pipelinewise/cli/config.py

```diff
@@ -54,7 +63,7 @@ def from_yamls(cls, config_dir, yaml_dir='.', vault_secret=None):
             target_id = target_data['id']
 
             # Check if a target with same ID already exists
-            # exit with error, otherwise, we would override the previous one and that would go unnoticed.
+            # exit with error, oterwise, we would override the previous one and that would go unnoticed.
```

Comment:
`otherwise` is the correct one :lel:

Reply:
fixed by https://github.com/transferwise/pipelinewise/pull/467/commits/0ebaab95c33cce5ac03d60ccd9e90dad7c6c9e82